### ISSUE TITLE
Odroid XU4 stuck on block 2783022

### ISSUE THREAD

**Issue by marcus290** [navcoin/navcoin-core#414]
Many Odroid XU4 users are stuck syncing on block 2783022. After bootstrapping from block 2778542, their wallets cannot `ConnectBlock()` on 2783023, often with the error :
```
ERROR: IsValidPaymentRequest: Could not find parent proposal 6e7935a0287d7ca1b9d44987c079872a01a82aa85cbddf356e0711d281a1c266 for payment request 02e2480265319d8274279bfeeefb8b81d35d8b4f0407b4dd7c77a3e86404b3e0
```
Restarting the wallet or attempting to re-bootstrap again does not fix this, and there are further errors logged:
```
CountVotes: Can't find block 763cb002854c6099c56e47897df71e1f8637d32e740776d72407ab47110e68f1 of proposal d0e4881e311508981f605a5f698fde932d8a83fe6d854ecf7c1886d19b344996
CountVotes: Can't find block 7882ce2a08406cca50a45b97f8a7cdba1de2adc9766266abb262dce89932f0d8 of proposal 4206f890851a65cc4e15098a63732ee45ebb6c56f3c13104e9e7921a1e5f87ff
CountVotes: Can't find block 2718fe6d498e9d42a748a6da170f8c80a04942a551f13822af4be4906254751f of proposal 2da711320aa9e83b91f6e4111898abc0214fdb347623a5b2286d3caf33c1e8dd
...
CountVotes: Can't find block fd072a0980e2443f5314abe67f06104645bf2b6ed9fe805924e3fbbabd2a85c1 of proposal 1bfe004a6329b64d59f3d2dc9e0ee7cd049cc986faced30f5d27ef1785e38c1d
```
These errors indicate problems accessing cfund data from `mapBlockIndex` and possibly `pblocktree` which are global variables initialized on wallet startup.

---
I personally only experienced this issue once, and the wallet successfully synced when bootstrapping again. I'm running:
- Ubuntu 18.04 (ubuntu-18.04.1-4.14-mate-odroid-xu4-20181203.img.xz
 from https://wiki.odroid.com/odroid-xu4/os_images/linux/ubuntu_4.14/ubuntu_4.14)
- `navcoind` v4.5.1 from the terminal (navcoin-4.5.1-arm-linux-gnueabihf.tar.gz from https://github.com/NAVCoin/navcoin-core/releases)
- tried bootstrapping manually by unzipping the files and moving them to `.navcoin4` folder, and also running the daemon with `./navcoind -bootstrap=https://s3.amazonaws.com/navcoin-bootstrap/bootstrap-navcoin_mainnet.tar`
- total memory 2GB (no swap)
- 16GB microSD

**Comment by marcus290** [comment]
![screenshot at 2019-02-06 22-32-55](https://user-images.githubusercontent.com/36244342/52378346-56ead100-2a5f-11e9-8c13-3f18e4180143.png)
the specs above are syncing past the problem block

**Comment by mxaddict** [comment]
I think this issue was fixed in the 4.7.3 release (The stability testing would have found it if it was still an issue)
